apps/web/src/app/admin-console/organizations/members/members.component.ts

```diff
@@ -102,23 +105,24 @@ export class MembersComponent extends BaseMembersComponent<OrganizationUserView>
   memberTab = MemberDialogTab;
   protected dataSource = new MembersTableDataSource();
 
-  organization: Organization;
-  status: OrganizationUserStatusType = null;
+  organization: Signal<Organization | undefined>;
+  status: OrganizationUserStatusType | undefined;
   orgResetPasswordPolicyEnabled = false;
-  orgIsOnSecretsManagerStandalone = false;
 
-  protected canUseSecretsManager$: Observable<boolean>;
-  protected showUserManagementControls$: Observable<boolean>;
+  protected canUseSecretsManager: Signal<boolean> = computed(
+    () => this.organization()?.useSecretsManager ?? false,
+  );
+  protected showUserManagementControls: Signal<boolean> = computed(
+    () => this.organization()?.canManageUsers ?? false,
+  );
+  private refreshBillingMetadata$: BehaviorSubject<null> = new BehaviorSubject(null);
+  protected billingMetadata$: Observable<OrganizationBillingMetadataResponse> | undefined;
```

Comment:
This is assigned in the constructor, so no need to handle undefined:

```suggestion
  protected billingMetadata$: Observable<OrganizationBillingMetadataResponse>;
```

Consuming code should also be updated to remove nullish handling.